File: app/src/main/java/com/shua/likegank/data/entity/Content.java
package com.shua.likegank.data.entity;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Android Entity
 * Created by ShuangHua on 2017/5/2.
 */

public class Content extends RealmObject {

    @PrimaryKey
    public String url;
    public String content;

    public Content() {
    }

    public Content(String content, String url) {
        this.content = content;
        this.url = url;
    }
}

File: app/src/main/java/com/shua/likegank/data/entity/Content.kt
package com.shua.likegank.data.entity

import io.realm.kotlin.types.RealmObject
import io.realm.kotlin.types.annotations.PrimaryKey

open class Content : RealmObject {
    @PrimaryKey
    var url: String = ""
    var content: String = ""

    constructor()

    constructor(content: String, url: String) {
        this.content = content
        this.url = url
    }
}